cpu/schbench.py.data/schbench.yaml

```diff
@@ -1,25 +1,31 @@
 schbench_url: 'https://git.kernel.org/pub/scm/linux/kernel/git/mason/schbench.git'
-perf:
+perf_stat: !mux
     default:
-        perfstat: '-a'
+        perf_stat: ''
 pin: !mux
     default:
-        taskset: '0'
+        taskset: ''
+locking: !mux
+    default:
+        locking: True
 threads: !mux
     default:
-        num_threads: 10
+        num_threads: 2
 workers: !mux
     default:
         num_workers: 10
 bytes: !mux
     default:
-        bytes: 1000
-duration: !mux
-    default:
-        runtime: 100
-cpu_duration: !mux
+        bytes: 0
+runtime: !mux
     default:
-        cputime: 10000
-mode:
+        runtime: 10
+autobench: !mux
     default:
         autobench: False
+rps: !mux
+    default:
+        rps: 0
+num_operations: !mux
+    default:
+        num_operations: 5
```

Comment:
issue with the end of line, please take care of this

Reply:
done